modules/eks/eks-without-spotinst/variables.tf

```diff
@@ -0,0 +1,346 @@
+variable "region" {
+  type        = string
+  description = "AWS Region"
+}
+
+variable "availability_zones" {
+  type        = list(string)
+  description = "AWS Availability Zones in which to deploy multi-AZ resources"
+}
+
+variable "availability_zone_abbreviation_type" {
+  type        = string
+  description = "Type of Availability Zone abbreviation (either `fixed` or `short`) to use in names. See https://github.com/cloudposse/terraform-aws-utils for details."
+  default     = "fixed"
+  validation {
+    condition     = contains(["fixed", "short"], var.availability_zone_abbreviation_type)
+    error_message = "The availability_zone_abbreviation_type must be either \"fixed\" or \"short\"."
+  }
+}
+
+variable "managed_node_groups_enabled" {
+  type        = bool
+  description = "Set false to prevent the creation of EKS managed node groups."
+  default     = true
+}
+
+variable "oidc_provider_enabled" {
+  type        = bool
+  description = "Create an IAM OIDC identity provider for the cluster, then you can create IAM roles to associate with a service account in the cluster, instead of using kiam or kube2iam. For more information, see https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html"
+}
+
+variable "cluster_endpoint_private_access" {
+  type        = bool
+  default     = false
+  description = "Indicates whether or not the Amazon EKS private API server endpoint is enabled. Default to AWS EKS resource and it is `false`"
+}
+
+variable "cluster_endpoint_public_access" {
+  type        = bool
+  default     = true
+  description = "Indicates whether or not the Amazon EKS public API server endpoint is enabled. Default to AWS EKS resource and it is `true`"
+}
+
+variable "cluster_kubernetes_version" {
+  type        = string
+  default     = null
+  description = "Desired Kubernetes master version. If you do not specify a value, the latest available version is used"
+}
+
+variable "public_access_cidrs" {
+  type        = list(string)
+  default     = ["0.0.0.0/0"]
```

Comment:
I know this is probably "how it is currently", but this seems like a bad default. We should probably default to the more secure option of not allowing any access and adding on as needed.

Reply:
Defaulted to []